mods/d2k/maps/atreides-02a/map.yaml

```diff
@@ -0,0 +1,220 @@
+MapFormat: 7
+
+RequiresMod: d2k
+
+Title: Atreides 02a
+
+Description: Infiltrate the Imperial Basin and build up our forces until they are strong enough to eradicate the local Harkonnen presence.\n\nThe Harkonnen are reinforceing their troops by air, so be on your guard. Use the Outpost's radar to detect attacks from unexpected quarters.\n\nBe careful when minig the Spice. Spice mounds grow out of the sand. While a vital source of Spice, Spice mounds can damage or destroy any unit that blunders into them.\n\nGood luck.
```

Comment:
reinforceing -> reinforcing
I believe this is the correct form, google agrees too.